src/viem/preparedTransactionSerialization.ts

```diff
@@ -0,0 +1,65 @@
+// Helper functions making prepared transactions serializable
+// so they can be used in redux actions (or even stores).
+import { TransactionRequestCIP42 } from 'node_modules/viem/_types/chains/celo/types'
+
+const bigIntProps = ['value', 'gas', 'maxFeePerGas', 'maxPriorityFeePerGas', 'gatewayFee'] as const
```

Comment:
I used a known list for now. Because we need to know which properties to convert back from `string` to `bigint` in `getPreparedTransactions`.
This is fine as TypeScript will error if there's another `bigint` not listed.
The alternative was to store the keys converted to `string`. But maybe we'll eventually need that, once we use a more generic `TransactionRequest` for both Celo and Ethereum?